src/javascript/EditPanel/EditPanelContent/FormBuilder/Section/FieldSet/Field/SelectorTypes/Picker/PickerDialog/LeftPanel.spec.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import {shallowWithTheme} from '@jahia/test-framework';
+import {dsGenericTheme} from '@jahia/design-system-kit';
+
+import {LeftPanel} from './LeftPanel';
+
+describe('PickerDialog - LeftPanel', () => {
+    let props;
+    beforeEach(() => {
+        props = {
+            field: {
+                selectorOptions: [{
+                    name: 'A',
+                    value: 'site'
+                }]
+            },
+            site: 'digitall',
+            siteNodes: [],
+            lang: 'fr',
+            nodeTreeConfigs: [{}],
+            onSelectSite: jest.fn(),
+            selectedPath: '/selectedPath',
+            setSelectedPath: jest.fn()
+        };
+    });
+
+    it('should display siteSwitcher when there is multiple site to show', () => {
```

Comment:
Minor: the message of the case it seems not a good fit, maybe we should updated it to understand the case that we talk about the special case when we hide/show the site switcher component.

Reply:
I confess that I don't understand well the behaviour of this.  Can you propose a better message as you are the one that implement it ? :)

Reply:
should display siteSwitcher when the picker is not of type "site picker"